app/src/main/java/com/github/se/wanderpals/MainActivity.kt

```diff
@@ -148,8 +149,20 @@ class MainActivity : ComponentActivity() {
     }
   }
 
+  override fun onNewIntent(intent: Intent) {
```

Comment:
Maybe you can leave a documentation for this function for better readability, but I think this is not strictly necessary.